src/interfaces/alteration-interfaces/court-order-interface.ts

```diff
@@ -1,5 +1,6 @@
 export interface CourtOrderIF {
   fileNumber: string,
   orderDate?: string,
-  effectOfOrder?: string
+  effectOfOrder?: string,
```

Comment:
What is the getter for this? It almost looks like it is `getCourtOrderNum`...

Reply:
This could just be a static value string value.. though i'm not really sure the purpose.

Reply:
To also comment on the DATE, i inquired with Scott about this and he said there was no Date requirement in this component. 🤷

Reply:
I see a lack of symmetry between the IF fields and the getters/setters.

What does getCourtOrderNum return then?

Reply:
Ohhhhhhhhhhhhhhhh... `getCourtOrderNum` would make more sense if it was renamed `getFileNumber` (or getCourtOrderFileNumber) (or just have 1 getter for the whole object and then extract fields as needed).

Reply:
There is a definite disconnect between what the UI designs call it and how the back end references it. 
I'm not sure which way to go here. 
ie the UI calls this the Court Order Number where the back end calls it the `fileNumber`. 
So your preference is renaming everything to `fileNumber`. 
This would involve another PR to shared components as well..

Reply:
Argh.

How about adding a comment to fileNumber indicating that it's aka courtOrderNum (and maybe a similar comment for the getter/setter)?

Reply:
Will do!

What are your thoughts on this though, because this isn't the first time. Do we prefer there to be a bit of a translation between property names between ui / api, or would we have the translation at the point of display?

Reply:
I prefer API and UI to be in sync... imagine there's a schema in place... and the template can label it whatever it wants (which is more likely to change than the data structures anyway).

It's good to not incur a bunch of tech debt, but I also don't want to create a ton of work for you :)

Reply:
ah i see what you mean! Well, i can change everything here to fileNumber. Will require another update to POA Shared component in the future to update the prop name.